config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -47,13 +47,22 @@ postsubmits:
         - cip
         - --thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
         - --confirm
+        # TODO: these are *not* threads 🤷‍♂️
+        # https://www.geeksforgeeks.org/golang-goroutine-vs-thread/
+        #
+        # this should probably be a multiple of cores below
+        - --threads=14
```

Comment:
So technically go's scheduler also needs to spend some time on bookkeeping etc (think GC).

But the default we have in CIP is 10, which doesn't make much sense, especially when we have limited to 2 cores.

14 is probably a reasonable starting point at 7 cores.

Reply:
The number I set is arbitrary because mainly, the promoter is waiting for network operations when talking to the registry and sigstore so this was more of an attempt to open more communication channels. But I agree that it makes more sense to adjust the number to the number cores.

Reply:
Yeah -- Even with waiting for things, available network I/O scales with CPU cores typically, unless we're bound by some API rate limit we should at least roughly scale with CPU allocated.

Reply:
2 is a arbitrary starting multiple, we may want more or less

Reply:
we should see how this actually performs soon 😅